### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by catngh**
**What does this PR do?**

- Update readme, ref: https://www.notion.so/console-labs/mochi-api-c535fe8f67194a5288301c91a041147b

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,36 +1,178 @@
...
+**Media** (Demo) (optional)
+Attach images or demo videos
+Can insert video URL in case the size exceeds github limit
 ```
+
+## Technical Document
+
+1. **Project structure**
+
+- `cmd/`
```
Comment (nguyennh4522) [inline README.MD:107]: should have a description for `cmd`, this folder contains the main application entry point files for the project, with the directory name matching the name for the binary

Comment (nguyennh4522) [reply README.MD:107]: I think you should read about golang standard structure

File: README.MD [hunk 2]
```diff
@@ -1,36 +1,178 @@
...
+- `docs/`: contains Swagger documentation files generated by `[swaggo](https://github.com/swaggo/swag)`
+- `migrations/`
+  - `schemas/`: contains DB schema migration files
+  - `seed/`: contains seed files which will initialize DB with sets of dummy data
+  - `test-seed/`: also seed files but for test DB
+- `pkg/`: contains core source code of service
+
+2. **Modules references**
+
+[Invite tracker](https://www.notion.so/Invite-tracker-40503e89ce40437c957f1eba1c87905d)
```
Comment (nguyennh4522) [inline README.MD:119]: there should be a tab before the content in an item

File: README.MD [hunk 3]
```diff
@@ -1,36 +1,178 @@
...
+
+If you'd like to contribute, please check out the [contributing guide](CONTRIBUTING.md).
+
+[![Contributing](https://contrib.rocks/image?repo=consolelabs/mochi-api)](https://github.com/consolelabs/mochi-api/graphs/contributors)
+
+## Keep in touch
+
+- Reach us at [discord](https://discord.gg/dddsYkB8Jw).
+- Discuss development in the #build-with-us channel.
+- [Core team members and Contributors](https://www.notion.so/3ab4b42bbc564e7b8d767521c25e78a0).
```
Comment (nguyennh4522) [inline README.MD:178]: this is not public link

File: .env.sample [hunk 4]
```diff
@@ -18,6 +18,7 @@ TWITTER_ACCESS_TOKEN=''
 TWITTER_ACCESS_TOKEN_SECRET=''
 
 IN_DISCORD_WALLET_MNEMONIC='tag volcano eight thank tide danger coast health above argue embrace heavy'
+CENTRALIZED_WALLET_MNEMONIC='tag volcano eight thank tide danger coast health above argue embrace heavy'
```
Comment (trkhoi) [inline .env.sample:21]: use this
```CENTRALIZED_WALLET_PRIVATE_KEY='account metamask private IN_DISCORD_WALLET_MNEMONIC='metamask recovery phrase'```

File: README.MD [hunk 5]
```diff
@@ -1,36 +1,178 @@
...
+   ```
+   make setup
+   ```
+
+2. Set up env
+
+   Update these variables inside `.env`, [How to get mnemonic](https://metamask.zendesk.com/hc/en-us/articles/360015290032-How-to-reveal-your-Secret-Recovery-Phrase)
+
+   ```
+   CENTRALIZED_WALLET_MNEMONIC=“your metamask wallet mnemonic”
```
Comment (trkhoi) [inline README.MD:55]: CENTRALIZED_WALLET_PRIVATE_KEY

**Review by trkhoi — CHANGES_REQUESTED** [review]

**Review by trkhoi — APPROVED** [review]

File: README.MD [hunk 6]
```diff
@@ -1,36 +1,192 @@
...
+**Media** (Demo) (optional)
+Attach images or demo videos
+Can insert video URL in case the size exceeds github limit
 ```
+
+## Technical Document
+
+1. **Project structure**
+
+- `cmd/` this folder contains the main application entry point files for the project,
```
Comment (nguyennh4522) [inline README.MD:107]: remove `,`

File: README.MD [hunk 7]
```diff
@@ -1,36 +1,192 @@
...
+  - `server/main.go`: Containing init function `main()` of the service. This function will be executed when we run `make dev`
+  - Other `main.go` files (e.g. `fetch-discord-users/main.go`): Entry points to execute some functional jobs. k8s Cronjob will be set up to execute these jobs schedually
+- `docs/`: contains Swagger documentation files generated by [swaggo](https://github.com/swaggo/swag)
+- `migrations/`
```
Comment (nguyennh4522) [inline README.MD:111]: add description for `migrations`

File: README.MD [hunk 8]
```diff
@@ -1,36 +1,192 @@
...
+  - `schemas/`: contains DB schema migration files
+  - `seed/`: contains seed files which will initialize DB with sets of dummy data
+  - `test-seed/`: also seed files but for test DB
+- `pkg/`: contains core source code of service
+  - `cache/`: caching initial and functional methods
+  - `chain/`: crypto chains’ initial and functional methods
+  - `config/`: `.env` file’s loader and reader
```
Comment (nguyennh4522) [inline README.MD:118]: it's not just load and read `.env`. You should show what `config` is used for, not what it contains

**Review by nguyennh4522 — APPROVED** [review]

File: README.MD [hunk 9]
```diff
@@ -1,36 +1,192 @@
...
+
+- [Price checker](https://www.notion.so/Price-checker-4c7a9ddca67e485284451216e152be4c)
+
+- [NFT](https://www.notion.so/NFT-bc88b0a16fb7457d9621e35ddc5cc90d)
+
+- [NFT Role](https://www.notion.so/NFT-Role-174945895e164d65a2b0df4f262a3cb1)
+
+3. **Sample usecases**
+
+   1. Create new API
```
Comment (nguyennh4522) [inline README.MD:158]: its not match with ref

**Pull request merged**